CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
```

Comment:
Almost the same, but 2.0 is wider by two words.

Reply:
Which are "caste, color", nicely spot.

Reply:
I suspect that github must have pulled a not-quite-released version of this document for their template. I can update to the correct current version, though.